### PR TITLE
[ADF-4831] DocumentList - filetype alternative text is not meaningful

### PR DESCRIPTION
**Pull request by pionnegru**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
[ADF-4831](https://issues.alfresco.com/jira/browse/ADF-4831)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/datatable/components/datatable/datatable.component.html [hunk 1]
```diff
@@ -110,14 +110,16 @@
                                     <ng-template #no_selected_row>
                                         <img class="adf-datatable-center-img-ie"
                                             [attr.aria-label]="data.getValue(row, col) | fileType"
-                                            alt="{{ iconAltTextKey(data.getValue(row, col)) | translate }}"
+                                            alt="{{ 'ADF-DATATABLE.ACCESSIBILITY.ICON_ALT_TEXT' | translate:{ type: data.getValue(row, col) | fileType } }}"
```
Comment (DenysVuika) [inline lib/core/datatable/components/datatable/datatable.component.html]: are all the translations available for the file types?

Comment (pionnegru) [reply lib/core/datatable/components/datatable/datatable.component.html]: should be covered when [this](https://github.com/Alfresco/alfresco-ng2-components/pull/4998/files/35a116419ac078986712eb0c61bb6872026c4bda#diff-e43039c591eae18181382500e64ba7e8R112) was done I guess

Comment (DenysVuika) [reply lib/core/datatable/components/datatable/datatable.component.html]: can then that work be included into this PR? it does not make much sense splitting that into multiple PRs

Comment (eromano) [reply lib/core/datatable/components/datatable/datatable.component.html]: agree

Comment (eromano) [reply lib/core/datatable/components/datatable/datatable.component.html]: I agree with @DenysVuika

**Comment by DenysVuika** [comment]
@pionnegru I think it needs rebase

**Pull request merged**